subworkflows/local/bam_variant_calling_somatic_controlfreec/main.nf

```diff
@@ -25,18 +25,44 @@ workflow BAM_VARIANT_CALLING_SOMATIC_CONTROLFREEC {
 
     ch_versions = Channel.empty()
 
+    assess_significance_input = Channel.empty()
```

Comment:
I'm wondering if it is necessary to introduce `assess_significance_input`? 🤔

Could we perhaps skip it and instead do just `ASSESS_SIGNIFICANCE(tumor_files)`?

Reply:
Done